locale/en/docs/guides/simple-profiling.md

```diff
@@ -131,20 +131,37 @@ of a user request but even in our simple example, time could be lost compiling
 regular expressions, generating random salts, generating unique hashes from user
 passwords, or inside the Express framework itself.
 
-Since we ran our application using the --prof option, a tick file was generated
+Since we ran our application using the `--prof` option, a tick file was generated
 in the same directory as your local run of the application. It should have the
-form `isolate-0xnnnnnnnnnnnn-v8.log` (where `n` is a digit). In order to make
-sense of this file, we need to use the tick processor included in the Node.js
+form `isolate-0xnnnnnnnnnnnn-v8.log` (where `n` is a digit).
+
+
+### For Node.js 4.x
+
+In order to make sense of this file, we need to use the tick processor included
+in the Node.js
 source at `<nodejs_dir>/tools/v8-prof/tick-processor.js`. It is important that
 the version of the tick-processor comes from the source for the version of node
-used to generate the isolate file. (This will no longer be a concern when the
-tick processor is [installed by default][].) The raw tick output can be
+used to generate the isolate file. The raw tick output can be
 processed using this tool by running:
 
 ```
-node <path_to_nodejs_src>/tools/v8-prof/tick-processor.js isolate-0x101804c00-v8.log >processed.txt
+node <path_to_nodejs_src>/tools/v8-prof/tick-processor.js isolate-0xnnnnnnnnnnnn-v8.log > processed.txt
 ```
 
+
+### For Node.js 5.x
+
+Starting Node.js 5.2.2, the the tick processor which is bundled inside the Node.js
```

Comment:
Maybe: "Starting in Node.js 5.2.2, the tick processor is bundled with the Node.js binary and can be run using the `--prof-process` flag:"